meteor/server/api/blueprints/api.ts

```diff
@@ -75,16 +76,21 @@ export async function removeBlueprint(methodContext: MethodContext, blueprintId:
 	removeSystemStatus('blueprintCompability_' + blueprintId)
 }
 
+export interface UploadBlueprintOptions {
+	blueprintName?: string
+	ignoreIdChange?: boolean
+	developmentMode?: boolean
```

Comment:
These could use some descriptions, detailing the functionality and intended use?